integration_test/disclosure_session/special_scenarios/combined_session.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:irmamobile/src/screens/session/disclosure/widgets/disclosure_permission_share_dialog.dart';
+import 'package:irmamobile/src/screens/session/session_screen.dart';
+import 'package:irmamobile/src/screens/session/widgets/issuance_permission.dart';
+import 'package:irmamobile/src/widgets/credential_card/irma_credential_card.dart';
+import 'package:irmamobile/src/widgets/irma_button.dart';
+import 'package:irmamobile/src/widgets/irma_card.dart';
+
+import '../../helpers/helpers.dart';
+import '../../helpers/issuance_helpers.dart';
+import '../../irma_binding.dart';
+import '../../util.dart';
+
+Future<void> combinedSessionTest(WidgetTester tester, IntegrationTestIrmaBinding irmaBinding) async {
```

Comment:
We call this _combined disclosure-issuance sessions_ in the documentation. It would be nice if this name comes back here.

https://irma.app/docs/session-requests/#issuance-requests